.github/workflows/release.yml

```diff
@@ -21,11 +21,13 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKER_IMAGE: smallstep/step-cli
+      DEBIAN_VERSION: bullseye
     outputs:
       version: ${{ steps.extract-tag.outputs.VERSION }}
       vversion: ${{ steps.extract-tag.outputs.VVERSION }}
       is_prerelease: ${{ steps.is_prerelease.outputs.IS_PRERELEASE }}
       docker_tags: ${{ env.DOCKER_TAGS }}
+      docker_debian_tags: ${{ env.DOCKER_DEBIAN_TAGS }}
```

Comment:
you could make this docker_tags_debian to align with step-kms-plugin docker_tags_cloud

Reply:
Fixed with 9bd605c